gradle/spek.gradle

```diff
@@ -24,13 +24,10 @@ dependencies {
 
   testRuntimeOnly("org.junit.platform:junit-platform-launcher")
   testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine")
-  testRuntimeOnly("org.junit.vintage:junit-vintage-engine")
   testRuntimeOnly("org.jetbrains.spek:spek-junit-platform-engine")
   testImplementation("io.strikt:strikt-core")
 }
 
 test {
-  useJUnitPlatform {
-    includeEngines "spek", "junit-vintage", "junit-jupiter"
-  }
+  useJUnitPlatform()
```

Comment:
with `useJUnitPlatform()` in build.gradle, can't we get rid of it here?

Reply:
Removed the `useJUnitPlatform()`, still test coverage remain same (4479).